test/functional/eval/timer_spec.lua

```diff
@@ -220,16 +220,20 @@ describe('timers', function()
       :good^                                   |
     ]])
 
+    -- Wait for timer to have finished / echoed.
+    retry(nil, nil, function()
```

Comment:
Adding extra RPC requests causes extra redraws. This negates the purpose of the test: to check whether nvim redraws the command line after the timer redraw completely by itself.